src/twisted/protocols/test/test_tls.py

```diff
@@ -1819,3 +1833,53 @@ def acceptableProtocols(self):
         @rtype: L{list} of L{bytes}
         """
         return self._acceptableProtocols
+
+
+class AggregateSmallWritesTests(SynchronousTestCase):
+    """Tests for ``AggregateSmallWrites``."""
+
+    @given(
+        st.lists(
+            st.one_of(
+                st.none(),
+                st.integers(min_value=1, max_value=100_000).map(
+                    lambda length: (b"0123456789ABCDEFGHIJ" * ((length // 20) + 1))[
+                        :length
+                    ]
+                ),
+            ),
+            max_size=1_000,
+        )
+    )
```

Comment:
Adding a descriptive comment to help people unfamiliar with Hypothesis might be nice. e.g.:

`# "writes" is a list of proposed writes, each bytestrings 1 to 100k long or a None`

Also I _believe_ that `None` is used here as a sentinel to flush the output (and not to test some feature of the interface) -- so that could be clarified in a comment too.